### DISCUSSION TITLE
The extension is not working

### DISCUSSION THREAD

**Discussion by richardalger** [Authenticator-Extension/Authenticator#1212] [General]
A month ago everything was fine, and now when I need the code the extension does not open.
 
![Screenshot 2024-06-20 203821](https://github.com/Authenticator-Extension/Authenticator/assets/173424547/721b58ce-48f5-42ad-b683-079b223ae3b0)
![Screenshot 2024-06-20 203842](https://github.com/Authenticator-Extension/Authenticator/assets/173424547/cefc15f4-1c02-4067-85e8-e02130481b40)

**Comment by richardalger** [comment]
popup.js:2 Uncaught (in promise) TypeError: Cannot read property 'UserSettings' of undefined
    at Function.<anonymous> (popup.js:2)
    at popup.js:2
    at Object.next (popup.js:2)
    at a (popup.js:2)
DevTools failed to load source map: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/dist/popup.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/dist/argon.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/css/popup.css.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

**Comment by Sneezry** [comment]
Execute `chrome.storage.sync.set({UserSettings:{}});chrome.storage.local.set({UserSettings:{}});` in popup page console, and check if it helps.

BTW, what is your browser version? Have you tried to upgrade to the latest version?

**Reply by richardalger** [reply]
Hello, didn't help, I updated to the latest version of chrome.
![Screenshot 2024-06-25 132717](https://github.com/Authenticator-Extension/Authenticator/assets/173424547/a18e4381-cee9-431c-aad7-23865f8d6b05)

**Comment by mymindstorm** [comment]
Closing this in favor of  #1215
